OrbitCore/OrbitLib.cpp

```diff
@@ -52,9 +52,9 @@ void Orbit::Init(const std::string& a_Host) {
 void Orbit::InitRemote(const std::string& a_Host) {
   Init(a_Host);
 
```

Comment:
We can remove the commented code altogether, this will be cleaned up soon as the Windows version will use the same pattern as the Linux version for uniformity (having a UI instance and a service).